core/src/main/java/org/apache/accumulo/fate/Fate.java

```diff
@@ -268,18 +268,18 @@ public long startTransaction() {
 
   // start work in the transaction.. it is safe to call this
   // multiple times for a transaction... but it will only seed once
-  public void seedTransaction(long tid, Repo<T> repo, boolean autoCleanUp) {
+  public void seedTransaction(long tid, Repo<T> repo, boolean autoCleanUp, String goalMessage) {
     store.reserve(tid);
     try {
       if (store.getStatus(tid) == TStatus.NEW) {
         if (store.top(tid) == null) {
           try {
+            log.info("FATE TxId: {} {}", tid, goalMessage);
```

Comment:
Should these be debug instead of info? Info seems like it might be a bit noisy.

Reply:
Here is a comparison of other things we log in the Manger.
<pre>
2022-02-14T10:09:58,426 [manager.EventCoordinator] INFO : State changed from HAVE_LOCK to NORMAL
2022-02-14T10:09:58,467 [manager.Manager] INFO : New servers: [ip-10-113-12-25:9997[10000844c5c0005], ip-10-113-12-25:10000[10000844c5c0006]]
2022-02-14T10:09:58,475 [manager.EventCoordinator] INFO : There are now 2 tablet servers
2022-02-14T10:09:58,480 [manager.Manager] INFO : tserver availability check disabled, continuing with-2 servers. To enable, set manager.startup.tserver.avail.min.count
2022-02-14T10:09:58,555 [manager.Manager] INFO : Setting manager lock data to ip-10-113-12-25:9999
</pre>
<pre>
2022-02-14T11:00:56,338 [manager.Manager] DEBUG: Finished gathering information from 2 of 2 servers in 0.01 seconds
2022-02-14T11:00:58,224 [vfs.ContextManager] DEBUG: Managed Contexts: {}
2022-02-14T11:00:58,224 [vfs.ContextManager] DEBUG: Configured Contexts: []
2022-02-14T11:00:58,224 [vfs.ContextManager] DEBUG: Closing contexts: {}
</pre>

Reply:
I know users who do create and delete a lot of tables. :smiley_cat:

The way I have thought about logging levels for server-side stuff is: INFO is best used for information about the server's status, like startup / shutdown, and major lifecycle events. DEBUG is useful for troubleshooting operations whose status would normally be observed by some other mechanism. Example: the fact that a table no longer appears in the table listing and a successful execution of the API for deleting a table, are normally sufficient to know that a table has been deleted, but debug logs can show the table being deleted on the server side. TRACE is useful for detailed information that we normally don't want to see, even for troubleshooting, because it's too noisy or not useful, unless we're specifically tracking down a bug in specific code that we are developing.

Also, I think INFO and higher is stuff that you wouldn't mind seeing in STDOUT if you were to run the process in a terminal. After hours of running, you should still be able to scroll up to the first few lines of this output. DEBUG is stuff you'd want to go to a file for troubleshooting later.

Reply:
> would (tid=#) add clarity?

I am not sure. Your version has one more character so I was trying to keep the messages to the minimum. I was also trying to only log the TableId in parenthesis for consistency.

Reply:
> Here is a comparison of other things we log in the Manger.

Yeah, most of that is startup/initialization stuff reporting the status of the Manager. I think that's consistent with what I was suggesting above. That's the kind of stuff I'd like to see in STDOUT when I start a process and watch it in a terminal (stuff that would also be visible in the output of `systemctl status accumulo-manager.service` or similar).